wiki/Main_Page/en.md

```diff
@@ -3,35 +3,35 @@ layout: "main_page"
 ---
 
 <div class="wiki-main-page__blurb">
-Welcome to osu!, a free rhythm game created by peppy with its primary focus of clicking circles along with the beat. Aside from the main game mode, which is referred to many as *osu!standard*, osu! has three other different game modes, namely: osu!taiko, osu!catch, and osu!mania.
+Welcome to to the osu! knowledge base, or otherwise known as the osu!wiki. This is a resource for new and veteran players to use for osu!.
```

Comment:
you really shouldn't be writing prose. read this back to yourself. you're writing words for the sake of writing words, not to actually convey anything of value (focus on the second line)

Reply:
Reading back, yeah I think this was completely butchering the entire blurb and its not even delivering the message propery. Please rewrite this.